### DISCUSSION TITLE
Can my team only have 2 members?

### DISCUSSION THREAD

**Discussion by 2uanDM** [microsoft/RAG_Hack#34] [Q&A] [answered]
Hi guys, after reading the official rules of RAG Hack: 
"Teams: Each Team must be comprised of one (1) Team Leader and two (2) or more Team Members. Only one (1) Entry per Team may be submitted in the Contest; we will accept no more than one (1) Entry per Team"

I want to ask a question that, if my team only has 2 members, is that accepted ? Thank you for answering my question

**Comment by artemious7** [comment]
I have the same question. Also, can a team consist of 4 or more members? From the Definitions sections, it seems, not:
> “Team” and “Teams” refers to a group of up to **three** (3) Contest participants working together to produce a single entry. 

It looks like, it can be either 1 individual contributor or a team of exactly 3 members.

**Comment by multispark** [comment, marked as answer]
@2uanDM @artemious7 
Having a team of 2 people is fine, the Team Leader also counts as a team member.

Additionally, we updated the competition rules to allow teams of up to 4 team members. Just keep in mind that the prize amounts will not change based on how many team members you have.

**Comment by WajahatAliHassan** [comment]
Is it fine to go with solo development or not, for RAG hack?  @multispark

**Reply by multispark** [reply]
Yes, solo development adheres to competition guidelines!
